routes/handlers/facebook.js

```diff
@@ -1,9 +1,14 @@
 const Facebook = require('../../lib/facebook');
 
 module.exports.events = {
-  handler: function (request, reply) {
-    Facebook.getEvents(function(err, res) {
-      return reply(res).type('application/json');
-    });
-  }
+    handler: function (request, reply) {
+        Facebook.getEvents(function (err, res) {
+            return reply(res);
+        });
+    },
+    plugins: {
+        hal: {
```

Comment:
Ah, yep, this was me.  It was part of the Halacious plugin that I was trying to put in.  Let me remove that real quick.  I goofed up and didn't use a branch on my forked repo and messed this up.  Good catch.